src/collision/collider/world_query.rs

```diff
@@ -0,0 +1,33 @@
+#![allow(missing_docs)]
```

Comment:
I'd scope this to the `ColliderQuery` to make sure future additions to the file are not missing docs

Reply:
This is actually on purpose due to an issue with `QueryData`. Originally `ColliderQuery` was even in a different module, but I moved it to its own module so that I can make this `allow` more scoped.

The issue is that `QueryData` also auto-generates a public `ColliderQueryItem` type, which gives the `missing_docs` warning, and I'm not aware of a way to add this `allow` for it without targeting the whole module. I should probably add a comment that mentions this though.